### ISSUE TITLE
Small bug in Mermaid Support

### ISSUE THREAD

**Issue by zzxxhhzxh** [BoostIO/Boostnote#2326]
# Current behavior
I use the codes in bottom to create graph TD and gantt. When I input "```Mermaid", it is different from "```mermaid".
In "```Mermaid", the background of chart turns to the background of Editor, there will be a copy/paste icon on the right-top; In "```mermaid", the background  is what in Preview, no icon any more.

Moreover, the size change a lot for graph TD. In gantt, it changes a little but neigher of them are satisfactory, because of the font size, and the margin height is too much as three times of the height of chart.

Following is the codes I used:

```Mermaid
graph TD
A[Christmas] -->|Get money| B(Go shopping)
B --> C{Let me think}
C -->|One| D[Laptop]
C -->|Two| E[iPhone]
C -->|Three| F[fa:fa-car Car]
```
```Mermaid
gantt
title A GANTT Diagram
dateFormat YYYY-MM-DD
section A section
Completed task: done, des1, 2014-01-06, 2014-01-08
Active task: active, des2, 2014-01-09, 3d
Future task: des3, after des2, 5d
Future task2: des4, after des3, 5d

section Critical tasks
Completed task in the critical line: crit, done, 2014-01-06, 24h
Implement parser and jison: crit, done, after des1, 2d
Create tests for parser: crit, active, 3d
Future task in critical line: crit, 5d
Create tests for renderer: 2d
Add to mermaid: 1d

section Documentation
Describe gantt syntax: active, a1, after des1, 3d
Add gantt diagram to demo page: after a1, 20h
Add another diagram to demo page: doc1, after a1, 48h

section Last section
Describe gantt syntax: after doc1, 3d
Add gantt diagram to demo page: 20h
Add another diagram to demo page: 48h
```

# Expected behavior

# Steps to reproduce

1. 
2. 
3. 

# Environment

- Version :0.11.8
- OS Version and name :Windows10 home 64bit 1803

![image 002](https://user-images.githubusercontent.com/42593813/44473442-e3125480-a5f5-11e8-8110-97f68d52546b.jpg)
![image 001](https://user-images.githubusercontent.com/42593813/44473449-e73e7200-a5f5-11e8-9e79-f3e95ef0da49.jpg)

**Comment by zmcddn** [comment]
I experience the same problem, has this been fixed?

**Comment by DogfishBone** [comment]
The same problem. I think it's an **urgency** bug!

**Comment by Flexo013** [comment]
As far as I'm aware the strange sizes of rendered diagrams was solved in a recent build (0.15.0?), so I'm closing this issue (for now).

If you still experience this issue, then please comment here with an exact copy of the diagram text you put in and an image of how it was rendered. Then I'll reopen this issue.
